metadata/metadata_test.go

```diff
@@ -1243,9 +1277,102 @@ func expectedModels() ResourceTests {
 	}
 }
 
+func expectedUpdatedModels() ResourceTests {
+	return ResourceTests{
+		ModelTest{
+			Name:        "fraud",
+			Description: "fraud model",
+			Labels:      []NameVariant{},
+			Features:    []NameVariant{},
+			TrainingSets: []NameVariant{
+				{Name: "training-set", Variant: "variant"},
+			},
+		},
+		ModelTest{
+			Name:        "fraud",
+			Description: "fraud model",
+			Labels:      []NameVariant{},
+			Features:    []NameVariant{},
+			TrainingSets: []NameVariant{
+				{Name: "training-set", Variant: "variant"},
+				{Name: "training-set", Variant: "variant2"},
+			},
+		},
+		ModelTest{
+			Name:        "fraud",
+			Description: "fraud model",
+			Labels:      []NameVariant{},
+			Features:    []NameVariant{},
+			TrainingSets: []NameVariant{
+				{Name: "training-set", Variant: "variant"},
+				{Name: "training-set", Variant: "variant2"},
+			},
+		},
+	}
+}
+
+func modelUpdates() []ResourceDef {
```

Comment:
Should add some cases like adding a trainingset or feature twice

Reply:
Maybe I should comment this function to make it clearer, but I iterate over these items and call `CreateModel` for each, so the 2nd and the 3rd item do test for adding the same training set twice. Would you like me to add the same cases for features, too?